OpenRA.Mods.Common/Traits/Render/RenderHarvester.cs

```diff
@@ -16,7 +16,9 @@ namespace OpenRA.Mods.Common.Traits
 {
 	class RenderHarvesterInfo : RenderUnitInfo, Requires<HarvesterInfo>
 	{
+		/* [SequenceReference] TODO: lint rule does not support arrays */
```

Comment:
We shouldn't be in a hurry to fix this, because it is a bit hacky.  We could perhaps do this with different body rendering sprites that are switched using upgrades.